sdk/js/src/Client.ts

```diff
@@ -75,6 +75,14 @@ export class DVCClient implements Client {
                 this.eventEmitter.emitError(err)
                 return this
             })
+            
+        if (!options?.reactNative && typeof window !== 'undefined') {
+            window.addEventListener('message', (event) => {
+                if (event.data === 'DVC.optIn.saved') {
```

Comment:
should we update this and the opt-in widget to use the object structure?
```suggestion
                const message = event.data
                if (message?.type === 'DVC.optIn.saved') {
```

Reply:
yup, will make a PR to update the message it in the widget